setup.py

```diff
@@ -42,7 +42,7 @@ def get_version():
         "Programming Language :: Python :: 3",
         "Topic :: Scientific/Engineering :: Physics",
     ],
-    install_requires=["numba>=0.51.0", "scipy", "psutil", "qibo>=0.1.7rc0"],
+    install_requires=["numba>=0.51.0", "scipy", "psutil", "qibo>0.1.8"],
```

Comment:
Shouldn’t we use 0.1.9 because the import of npmatrices will fail with 0.1.8?

Reply:
Well, there is a `>` not `>=`, right?

Reply:
Right, thanks.